packages/core/resources/en/app.yaml

```diff
@@ -50,6 +50,7 @@ scriptLoadPost: Preparing script tasks
 scriptLoadModule: Loading script from configuration or node module
 scriptRun: Running {{name}} script
 signalBufferMessage: 'The following output was buffered before being interrupted:'
+stylelintDescription: Lint files with stylelint
```

Comment:
Maybe "Link CSS with stylelint"?

Reply:
what about "lint styles", since it can lint styles in lots of places of formats/files (scss, html, etc)

Reply:
Yeah thats chill